lib/agenda.js

```diff
@@ -112,15 +117,14 @@ Agenda.prototype.define = function(name, options, processor) {
 Agenda.prototype.every = function(interval, names, data) {
   var self = this;
 
-  if (typeof names === 'string') {
+  if (typeof names === 'string' || names instanceof String) {
```

Comment:
when would the first one evaluate false, but the second one true? Unless there is a good reason, I'd get rid of this second case.